lms/models/grading_info.py

```diff
@@ -69,3 +69,11 @@ class GradingInfo(BASE):  # pylint:disable=too-few-public-methods
     tool_consumer_info_product_family_code = sa.Column(sa.UnicodeText(), nullable=True)
     h_username = sa.Column(sa.UnicodeText(), nullable=False)
     h_display_name = sa.Column(sa.UnicodeText(), nullable=False)
+
+    def __repr__(self):
+        return "GradingInfo({kwargs})".format(
+            kwargs=", ".join(
+                f"{kwarg}={repr(getattr(self, kwarg))}"
+                for kwarg in self.__table__.columns.keys()
+            )
+        )
```

Comment:
Could this go in `lms.db.BaseClass` and benefit all of our models? It could easily not be grading info specific by using `self.__class__.__name__` in place of `GradingInfo`.

There's also `iter_columns` in there which looks like it might be helpful for this? Maybe not. If we had an `as_dict()` which returned the `kwargs`, and then `__repr__` used that it would be pretty neat.

Reply:
Yep, moved this onto the base class